h2o-py/h2o/model/metrics_base.py

```diff
@@ -1992,14 +2121,14 @@ def plot_uplift(self, server=False, save_to_file=None, plot=True, metric="auto")
         >>> uplift_model = H2OUpliftRandomForestEstimator(ntrees=10, 
         ...                                               max_depth=5,
         ...                                               treatment_column=treatment_column,
-        ...                                               uplift_metric="qini",
+        ...                                               uplift_metric="kl",
         ...                                               distribution="bernoulli",
-        ...                                               gainslift_bins=10,
         ...                                               min_rows=10,
         ...                                               auuc_type="gain")
         >>> uplift_model.train(y=response_column, x=predictors, training_frame=train)
-        >>> uplift_model.plot_uplift(plot=True)
-        >>> n, uplift = uplift_model.plot_uplift(plot=False)
+        >>> perf = uplift_model.model_performance()
+        >>> perf.plot_uplift(plot=True)
```

Comment:
Here I get:
```

~/h2o-3/_valenad-notebooks/../h2o-py/build/main/h2o/model/metrics_base.py in auuc(self, metric)
   1819             return self._metric_json['AUUC']
   1820         else:
-> 1821             assert metric in ['qini', 'lift', 'gain'], \
   1822                "AUUC metric "+metric+" should be 'qini','lift' or 'gain'."
   1823             return self._metric_json['auuc_table'][metric][0]

AssertionError: AUUC metric AUTO should be 'qini','lift' or 'gain'.
```

Seems that `auuc` cannot hande AUTO

Reply:
I am still getting this error :/

Reply:
Hopefully, it is working now. I added a uses-case that tests it. Thanks for checking it again. :)